stream_alert_cli/logger.py

```diff
@@ -33,3 +54,14 @@
     if logger.startswith('StreamAlert'):
         continue
     logging.getLogger(logger).setLevel(logging.CRITICAL)
+
+# Get a logging MemoryHandler with a default buffer size of 1000
+# We don't care about assigning a target to this handler since these logs
+# will not actually be written out to disk, etc
+LOG_ERROR_HANDLER = logging.handlers.MemoryHandler(1000)
+
+# Add a filter to suppress everything that is not an error
+LOG_ERROR_HANDLER.addFilter(SuppressNonErrors())
+
+# Add the MemoryHandler to the root logger to capture all logs in all loggers
+logging.getLogger().addHandler(LOG_ERROR_HANDLER)
```

Comment:
If this extra log handler is only used for unit tests, maybe we should only add it if the user is running tests? Instead of adding an extra in-memory logging handler for all StreamAlert operations